src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1175,8 +1195,21 @@ namespace winrt::TerminalApp::implementation
     // Arguments:
```

Comment:
I'd maybe update the doc comment with a little bit more explanation that this is called when tabs are being re-ordered and this handlers that as well.

Reply:
Fixed in next commit